starlight/templates/login_style.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+{% load i18n %}
+{% load staticfiles %}
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>
+        {% block title %}{% trans "Starlight" %}{% endblock %}
+    </title>
+
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.1.8/semantic.min.css" rel="stylesheet">
+
+    {% block extrahead %}{% endblock %}
+
+    <style type="text/css">
```

Comment:
For the time being this is fine but we will eventually want to move it to a dedicated CSS file that is bundled by Webpack :)